internal/post/sender.go

```diff
@@ -73,26 +77,31 @@ func (sender *SMTPSender) SendEmail(ctx context.Context, msg *Message) (err erro
 		}
 	}
 
-	data, err := client.Data()
+	mess, err := msg.Bytes()
```

Comment:
Why was this moved upward?

Reply:
for `data` to be closer to function that invokes it